modules/social_features/social_comment/src/Form/SocialCommentAdminOverview.php

```diff
@@ -184,7 +184,15 @@ public function buildForm(array $form, FormStateInterface $form_state, $type = '
     foreach ($comments as $comment) {
       // Get a render array for the comment body field. We'll render it in the
       // table.
-      $comment_body = $comment->field_comment_body->view('full');
+      if ($comment->hasField('field_comment_body')) {
+        $comment_body = $comment->field_comment_body->view('full');
+      }
+      elseif ($comment->hasField('comment_body')) {
+        $comment_body = $comment->comment_body->view('full');
+      }
+      else {
+        $comment_body = $this->t('n/a');
```

Comment:
I think this can probably just be an empty string as "n/a" isn't going to be helpful to site managers either.